conda_forge_tick/recipe_parser/_parser.py

```diff
@@ -106,12 +113,14 @@ def _parse_jinja2_variables(meta_yaml: str) -> dict:
                     assert False, jinja2_data
             else:
                 jinja2_vals[n.target.name] = (n.node.value, i)
+        elif isinstance(n, jinja2.nodes.Assign):
+            jinja2_exprs[n.target.name] = meta_yaml_lines[n.lineno-1]
```

Comment:
this breaks on the awscli recipe:
```
2020-03-28 04:23:48,125 INFO     conda_forge_tick.auto_tick || VERSION-Version IS MIGRATING awscli
2020-03-28 04:23:48,125 INFO     github3 || Building a url from ('https://api.github.com', 'rate_limit')
2020-03-28 04:23:48,156 INFO     github3 || JSON was returned
2020-03-28 04:23:48,156 INFO     github3 || Building a url from ('https://api.github.com', 'repos', 'conda-forge', 'awscli-feedstock')
2020-03-28 04:23:48,156 INFO     github3 || Missed the cache building the url
2020-03-28 04:23:48,256 INFO     github3 || JSON was returned
2020-03-28 04:23:48,256 INFO     github3 || Building a url from ('https://api.github.com', 'repos', '*********************', 'awscli-feedstock')
2020-03-28 04:23:48,257 INFO     github3 || Missed the cache building the url
2020-03-28 04:23:48,383 INFO     github3 || JSON was returned
~/repo/cf-graph/feedstocks/awscli-feedstock ~/repo/cf-graph
Already on 'master'
Your branch is up to date with 'origin/master'.
HEAD is now at a1fdeba Update to v1.14.45 (#27)
HEAD is now at 73ae868 Arch Migrator (#521)
error: pathspec '1.18.31' did not match any file(s) known to git
~/repo/cf-graph
2020-03-28 04:23:50,486 ERROR    conda_forge_tick.auto_tick || NON GITHUB ERROR
Traceback (most recent call last):
  File "/root/repo/cf-scripts/conda_forge_tick/auto_tick.py", line 922, in main
    hash_type=attrs.get("hash_type", "sha256"),
  File "/root/repo/cf-scripts/conda_forge_tick/auto_tick.py", line 166, in run
    migrate_return = migrator.migrate(recipe_dir, feedstock_ctx.attrs, **kwargs)
  File "/root/repo/cf-scripts/conda_forge_tick/migrators/version.py", line 397, in migrate
    cmeta = CondaMetaYAML(fp.read())
  File "/root/repo/cf-scripts/conda_forge_tick/recipe_parser/_parser.py", line 366, in __init__
    v, e = _parse_jinja2_variables(meta_yaml)
  File "/root/repo/cf-scripts/conda_forge_tick/recipe_parser/_parser.py", line 117, in _parse_jinja2_variables
    jinja2_exprs[n.target.name] = meta_yaml_lines[n.lineno-1]
AttributeError: 'Tuple' object has no attribute 'name'
```

same for boto3

Reply:
Ahhh thank you! I’ll push a fix

Reply:
https://github.com/regro/cf-scripts/pull/920